docs/general/builtins/scalar-functions.rst

```diff
@@ -4278,6 +4278,31 @@ Example:
     SELECT 1 row in set (... sec)
 
 
+.. _scalar-pg_table_is_visible:
+
+``pg_table_is_visible()``
+-------------------------
+
+The function ``pg_table_is_visible`` returns ``true`` if the current user holds
+``DQL`` privilege on the table and there are no other tables with the same
+table name(also with ``DQL``) but with different schema names appearing earlier
```

Comment:
Here you refer to `name` but we should clarify that the argument is only `oid`, you cannot use the name of the table/view as string.